src/v/cloud_topics/reconciler/reconciliation_consumer.cc

```diff
@@ -35,8 +34,6 @@ reconciliation_consumer::operator()(model::record_batch batch) {
 
     // NOTE: Only data batches here. It's safe to use timestamps without
     //       checking the batch type.
-    _metadata.base_timestamp = std::min(
-      batch.header().first_timestamp, _metadata.base_timestamp);
```

Comment:
why is base timestamp being removed (its removal is in the commit which renames partition -> source, so the commit reflects that not this removal which is probably misplaced?)

Reply:
No worries